### DISCUSSION TITLE
Add a base face for all things magit?

### DISCUSSION THREAD

**Discussion by DivineDominion** [magit/magit#4639] [Ideas]
I'm using `text-scale-decrease` once in a hook for magit; my default font is a bit taller for comfortable typing, but for magit's UI I find a smaller font closer to the OS font to work better.

But I feel dirty for using this hook :)

Would it make sense to add a `magit` face that all other faces inherit from? It wouldn't have to do anything, but then users (like me) could target all magit faces and adjust e.g. the height or the monospace/variable pitch setting at once.

What do you think?

**Comment by tarsius** [comment]
Keep using `text-scale-decrease` (or whatever it does at a low-level). Adding more inheritance to Magit's faces would mean more work for me, regressions for other users and a buggier experience for you.

**Reply by DivineDominion** [reply]
Alright, wasn't sure what would be the best composition.
